packages/cli/test/utils/crucible/clients/beacon/lighthouse.ts

```diff
@@ -88,9 +88,11 @@ export const generateLighthouseBeaconNode: BeaconNodeGenerator<BeaconClient.Ligh
       },
       health: async () => {
         try {
-          await got.get(`http://127.0.0.1:${ports.beacon.httpPort}/eth/v1/node/health`);
+          const response = await fetch(`http://127.0.0.1:${ports.beacon.httpPort}/eth/v1/node/health`);
+          // Lighthouse health endpoint returns 200 or throws, no need to check response.ok
```

Comment:
In this case, you don't need to define `response` variable at all as it is unused.

Reply:
Yes, true. 
Thanks for pointing it out.